Jobsway2goMvc/Models/JobCategory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Jobsway2goMvc.Models
+{
+    public class JobCategory
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public int JobId { get; set; }
+      //  public Job Job { get; set; }
+    }
```

Comment:
`List<Category> Categories` instead of JobId and Job. Category can't be one-to-one relationship as it holds more than one job per category.